src/components/Button/Button.module.scss

```diff
@@ -111,36 +114,100 @@
     border-color: var(--Transparent--white);
     background-color: var(--GrayStrokeAndDisabled--translucent);
     color: var(--White);
+
     path {
       fill: var(--White);
     }
   }
 }
 
-.Button.BlackOutline {
-  border-color: var(--GrayPrimary--opaque);
+.Button.BtnRoundedBlack {
+  border-radius: 8px;
+  background-color: var(--GrayPrimary--translucent);
+  color: var(--White);
+
+  path {
+    fill: var(--White);
+  }
+
+  @include light-on-dark-font-smoothing;
+
+  &:not([disabled]) {
+    &:hover {
+      background-color: var(--GrayDarkHover--translucent);
+    }
+
+    &:active {
+      background-color: var(--GrayDarkHover--translucent);
+    }
+  }
+
+  &[disabled] {
+    border-color: var(--Transparent--white);
+    background-color: var(--GrayStrokeAndDisabled--translucent);
+    color: var(--White);
+
+    path {
+      fill: var(--White);
+    }
+  }
+}
+
+.Button.BtnRoundedOutline {
+  border-radius: 8px;
+  border-color: #ccc;
+  font-weight: 500;
   background-color: var(--Transparent--white);
   color: var(--GrayPrimary--opaque);
+
   path {
     fill: var(--GrayPrimary--opaque);
   }
 
   &:not([disabled]) {
-    &:hover {
-      border-color: var(--GrayDarkHover--translucent);
-      background-color: var(--GrayDarkHover--translucent);
-      color: var(--White);
+    &:hover,
+    &:active {
+      /* Keep same as default */
+      border-color: #ccc; 
+      background-color: var(--Transparent--white);
+      color: var(--GrayPrimary--opaque);
+
       path {
-        fill: var(--White);
+        fill: var(--GrayPrimary--opaque);
       }
     }
+  }
+
+  &[disabled] {
+    border-color: var(--Transparent--white);
+    background-color: var(--GrayStrokeAndDisabled--translucent);
+    color: var(--White);
+
+    path {
+      fill: var(--White);
+    }
+  }
+}
+
+.Button.BlackOutline {
+  border-color: #ccc;
+  background-color: var(--Transparent--white);
+  color: var(--GrayPrimary--opaque);
+  font-weight: 500;
+  path {
+    fill: var(--GrayPrimary--opaque);
+  }
```

Comment:
Seems like we're changing some existing styles here, have these been tested for backwards compatibility for non-experiment versions of the button & navbar?